ArticleTemplates/assets/js/bootstraps/common.js

```diff
@@ -59,6 +60,29 @@ function init(liveBlog = false) {
     }
 }
 
+function setupArticlePlayback() {
+    // Called by native code
+    window.setupArticlePlayback = addPlaceholderToArticle;
+}
+
+function addPlaceholderToArticle() {
+    const placeholder = document.createElement('div');
+    // we put the audio player just below the standfirst
+    const listenToArticleSibling = document.querySelector('.standfirst');
```

Comment:
@waisingyiu so following our meeting with Luke, i guess here instead of querying the `standfirst` we need to find another suitable element to query so that we can place it above the quadrule